local-tests/tests/testPkpEthersWithEoaSessionSigsToSignWithAuthContext.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from '@lit-protocol/auth-helpers';
+import { PKPEthersWallet } from '@lit-protocol/pkp-ethers';
+import { AuthCallbackParams, AuthSig } from '@lit-protocol/types';
+import { TinnyEnvironment } from 'local-tests/setup/tinny-environment';
+
+/**
+ * Test Commands:
+ * ✅ NETWORK=cayenne yarn test:local --filter=testPkpEthersWithEoaSessionSigsToSignWithAuthContext
+ * ✅ NETWORK=manzano yarn test:local --filter=testPkpEthersWithEoaSessionSigsToSignWithAuthContext
+ * ✅ NETWORK=localchain yarn test:local --filter=testPkpEthersWithEoaSessionSigsToSignWithAuthContext
+ */
+export const testPkpEthersWithEoaSessionSigsToSignWithAuthContext = async (
+  devEnv: TinnyEnvironment
+) => {
+  const alice = await devEnv.createRandomPerson();
+
+  const pkpEthersWallet = new PKPEthersWallet({
+    pkpPubKey: alice.pkp.publicKey,
+    litNodeClient: devEnv.litNodeClient,
+    authContext: {
```

Comment:
What is the authContext here?

Reply:
The `authContext` allows you to pass the same parameters to `getSessionSigs` but also require you to pass in `litNodeClient` so that it picks the correct connection configure etc. It's confusing and needs some love, it's working but that's why i didn't put that in the docs. We will come back to this in next sprint